textproc/extractpdfmark/Portfile

```diff
@@ -27,23 +27,21 @@ checksums           rmd160  1f1bac62afac8b52e54a796084d5b6c87453122e \
                     sha256  63f3ababd5b1081ef92ff7a417597302327c1db3902cdb9827fade147558e6db \
                     size    295003
 
+depends_build       port:pkgconfig
```

Comment:
When dependencies or installed files change you need to increase the revision of the port, in this case add:
```
revision 1
```
(somewhere below the version number).

Reply:
Done.

Reply:
Thanks. Can you please change the documentation files just in case?

After removing the complete `post-destroot` block here's the list of files I get:
```
/opt/local/bin/extractpdfmark
/opt/local/share/man/man1/extractpdfmark.1.gz
/opt/local/share/doc/extractpdfmark/README.md
/opt/local/share/doc/extractpdfmark/README.ja.md
/opt/local/share/doc/extractpdfmark/COPYING
```

(I'll wait for Ryan this time :)